de.mediathekview.MediathekView.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
It's a bug to support a Linux desktop and not ship a .desktop.

Reply:
I'm sorry, @hfiguiere, but this doesn't help me.  I have only so much time for this, so as I said I'd very much prefer to get the flatpak working and only aftwards start upstreaming changes, and as far as I understood @barthalion having these files in here is not a blocker.

Upstream may have a quite different point of view, or may simply not care, so it may take some time to get these things upstream, and I may not even succeed.

Hence I'd prefer if the review could focus on what's actually blocking this package.  And, on a personal note, I'd also appreciate a bit less condescending and passive-aggressive comments 🙂